src/gui/raisedmessagebox.h

```diff
@@ -44,10 +45,8 @@ class RaisedMessageBox final : public QMessageBox
     void showEvent(QShowEvent *event) override;
 
 private:
+    RaisedMessageBox() = default;
```

Comment:
Is it even needed?

Reply:
I think it was to hide the constructor in private. And I added `default` since there was no implementation in .cpp file.

Reply:
If this constructor isn't used you can omit such a declaration. There is no implicitly declared default constructor if at least one user defined constructor is present so you don't need to declare it just to make it private.

Reply:
OK, removed.